.changeset/big-dryers-report.md

```diff
@@ -0,0 +1,37 @@
+---
+'braid-design-system': minor
+---
+
+---
+updated:
+  - Box
+  - BoxRenderer
+  - atoms
+  - vars
+---
+
+**Box, atoms, vars:** Add `small` to border radius scale
+
+Extends the border radius scale to include `small` as a step below `standard`.
+This uplift is to support an upcoming design uplift that requires greater fidelity in the scale.
```

Comment:
Feels weird to use `uplift` twice here`. IMO this is just an addition to the existing scale, rather than an uplift.
```suggestion
This addition is to support an upcoming design uplift that requires greater fidelity in the scale.
```